### PR TITLE
Update Travis Config to fix Postgres connection issue

### LINKED ISSUE #230: Travis CI build cannot connect to database
When Travis builds this project, rake cannot connect to postgres. Since no config/database.yml is found, `rake bootstrap` copies the example one which is not set up for travis.

### PR DESCRIPTION
**Pull request by YaxelPerez**
This PR closes #230

### CODE REVIEW

**Comment by YaxelPerez** [comment]
@xmunoz done

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,17 @@ cache: bundler
 rvm:
 - 2.2.3
 bundler_args: --without development
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install g++ qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
 before_script:
-- bundle exec rake bootstrap:db_config db:test:prepare
+  - psql -c 'create database travis_ci_test;' -U postgres
+  - cp config/database.yml.travis config/database.yml
```
Comment (xmunoz) [inline .travis.yml]: Please update file name here.

File: config/database.yml.example [hunk 2]
```diff
@@ -10,6 +10,6 @@ development:
 # Do not set this db to the same as development or production.
 test:
   adapter: postgresql
-  database: arcdata-test
+  database: travis_ci_test
```
Comment (xmunoz) [inline config/database.yml.example:13]: Instead of change the database name here, why not change the one in .travis.yml to `arcdata-test`?

Comment (YaxelPerez) [reply config/database.yml.example]: oops, yeah. Of course. Sorry

**Comment by xmunoz** [comment]
It looks like `cp` command was removed, but strangely the build doesn't seem affected. I looked in the build log, and travis appears to be doing the right thing by default:

```
$ bundle exec rake bootstrap:db_config db:test:prepare
/home/travis/build/redcross/arcdata/vendor/bundle/ruby/2.2.0/gems/ruby-ole-1.2.11.7/lib/ole/types/base.rb:265: warning: duplicated key at line 266 ignored: 4095
ls: cannot access 'config/database.yml': No such file or directory
-----> Installing example database config...
Executing: cp -n config/database.yml.example config/database.yml
```
Anyways, this looks ready for a merge now :)

**Comment by YaxelPerez** [comment]
> ```
> sudo apt-get install g++ qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
> ```

They're dependencies for capybara (https://github.com/thoughtbot/capybara-webkit/wiki/Installing-Qt-and-compiling-capybara-webkit#debian--ubuntu)

**Pull request merged**